tests/sentry/backup/test_models.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import atexit
+import tempfile
+from pathlib import Path
+
+from click.testing import CliRunner
+from django.core.management import call_command
+
+from sentry.db.postgres.roles import in_test_psql_role_override
+from sentry.runner.commands.backup import import_, validate
+from sentry.testutils import TransactionTestCase
+from tests.sentry.backup import ValidationError, tmp_export_to_file
+
+
+class ModelBackupTests(TransactionTestCase):
+    """Test the JSON-ification of models marked `__include_in_export__ = True`. Each test here
+    creates a fresh database, performs some writes to it, then exports that data into a temporary
+    file (called the "expected" JSON). It then imports the "expected" JSON and re-exports it into
+    the "actual" JSON file, and diffs the two to ensure that they match per the specified
+    comparators."""
```

Comment:
Also this seems reasonable to test import/export functionality but what if the model schemas change? Then it would break during load-it-up if I understand correctly, even though it passes these tests because it converts properly between import/export.

Reply:
We test all of the models written in a given test - basically `fresh-install.json` + whatever writes the specific test made.

Regarding schema changes, we discussed offline, but the plan is more or less what's documented at https://github.com/getsentry/team-ospo/issues/159. @kneeyo1 and I agreed that this would be a good way to guard against future schema skew. It's not ideal for testing against already-released schemas, though we're not sure what to do about that, except to maybe retroactively run the script mentioned in getsentry/team-ospo#159 retroactively against old releases.